### DISCUSSION TITLE
extension   #2

### DISCUSSION THREAD

**Discussion by Ali-Key** [sharafdin/markdown-master#9] [Q&A] [answered]
luuqadee ugu fiican inaa extension ku dhiso?

**Comment by miirshe** [comment, marked as answer]
waxad isticmali kartaa TypeScript aad ayey u fududahay hadii aad javascript taqaano sbb too ah safety programming language waaye 
waana static language oo wax walbo waad qeexi doontaa tusaale variable ayaad rabtaa inaad wax ku shubto waxa uu yahay ayaad sheegi dokntaa.
